### PR TITLE
docs(docs-infra): update where we are presenting

### PR DESCRIPTION
**Pull request by petebacondarwin**
Does what it says on the tin ^^^

### CODE REVIEW

File: aio/content/marketing/events.html [hunk 1]
```diff
@@ -44,18 +44,30 @@ <h1 class="banner-headline no-toc no-anchor">Events</h1>
...
-        <td>October 29-31, 2018</td>
+        <th><a href="https://ng-atl.org/" title="ngAtlanta">ngAtlanta</a></th>
+        <td>Atlanta, Georgia</td>
+        <td>January 9-12, 2019</td>
+      </tr>
+      <!-- ng-India 2019-->
+      <tr>
+        <th><a href="https://www.ng-ind.com/" title="ng-India">ng-India</a></th>
+        <td>Gurgaon, India</td>
+        <td>February 23, 2019</td>
```
Comment (gkalpak) [inline aio/content/marketing/events.html]: Based on the date, `ngIndia` should be above `ngAtlanta`.

**Review by gkalpak — APPROVED** [review]
LGTM, except I think upcoming events should be in chronological order.

**Review by StephenFluin — APPROVED** [review]

**Comment by gkalpak** [comment]
Shouldn't the upcoming events be in chronological order? 😞

**Comment by petebacondarwin** [comment]
Oh I didn't understand your previous comment @gkalpak - in my mind I was thinking - they are in chronological order latest at the top :-)

Feel free to fix this if you feel strongly about it. I don't.

**Pull request closed without merging**